### DISCUSSION TITLE
Does anyone know why Sumatra starts pdf with negative colour?

### DISCUSSION THREAD

**Discussion by martinkirlow** [sumatrapdfreader/sumatrapdf#4421] [General]
It started today and I don't know why it happened. I must have altered a menu or something but I don't why/what.

**Comment by GitHubRulesOK** [comment]
`i` key is invert

but could also be theming or an advanced  startup setting it should be simple to flip on and off
It may depend on version but inversion is one key brush away in recent versions !

<kbd>i</kbd> off/on in latest pre-release

![image](https://github.com/user-attachments/assets/d93fc8b5-ca4a-4301-87be-6cbf751226cc)![image](https://github.com/user-attachments/assets/6020ad6f-1bc2-4ca6-95c4-8f4c69742f90)

**Comment by kjk** [comment]
I just swapped key bindings: `i` is now `CmdTogglePageInfo`, `I` is `CmdInvertColors`. 

It's too easy to press `i` by accident.

**Reply by onebod** [reply]
Oh!  I happened to see in Advanced Options the word "InvertColors = true".
At least then I knew what term to search in this forum.
Not knowing HOW I inverted colours, simply changing the setting would have bugged me.
Thanks very much!

**Comment by martinkirlow** [comment]
Ah, thank you for that, both of you!

**Comment by jreynoldshrg** [comment]
omg lol that's all it was!? I was having the same issue.
